### PR TITLE
document upgrade process

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

**Comment by matklad** [comment]
bors r+
🤖

File: docs/user/README.md [hunk 1]
```diff
@@ -58,6 +58,9 @@ Beyond basic LSP features, there are some extension commands which you can
 invoke via <kbd>Ctrl+Shift+P</kbd> or bind to a shortcut. See [./features.md](./features.md)
 for details.
 
+For updates, pull the latest changes from the master branch, run `cargo install-ra` again, and **restart** VS Code instance.
```
Comment (bjorn3) [inline docs/user/README.md]: For me it required a restart between uninstall of the ext and `cargo install-ra`, otherwise it caches the old ext.

Comment (matklad) [reply docs/user/README.md]: Hm, do you mean you need to manually *uninstall* the extension as well? For me, it just overwrites the old extension...

Comment (DianaNites) [reply docs/user/README.md]: Never needed to uninstall the extension when upgrading either. Overwrites just fine.

Comment (bjorn3) [reply docs/user/README.md]: Just replacing it without uninstalling, seems to have worked this time.

**Pull request merged**